### PR TITLE
Hardening/117 ngsi9 subscription convops

### PR DESCRIPTION
**Pull request by kzangeli**
Last 'nec ngsi9 convops' converted to use standard ops.
Actually, they already did, just cleaned it up a little and added comments.

### CODE REVIEW

File: test/functionalTest/cases/117_convop_using_standard_ops/putAvailabilitySubscriptionConvOp.test [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+--NAME--
+ConvOp putAvailabilitySubscriptionConvOp: PUT /v1/registry/contextAvailabilitySubscriptions/{subscriptionId}
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
```
Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/putAvailabilitySubscriptionConvOp.test]: Should we involve NGSI9 notifications (and accumulator-server) in this tests? Or maybe it is not necessary? Not sure about that... just a "feeling"...

Comment (kzangeli) [inline test/functionalTest/cases/117_convop_using_standard_ops/putAvailabilitySubscriptionConvOp.test]: I was thinking about that too but as the purpose of this test is not to test the subscriptions themselves but the ability to create/modify them. We have to assume that if the broker says "ALL OK", all is ok :-)

What I would have liked here is to ask the broker for a listing of its subscriptions.

Comment (fgalan) [inline test/functionalTest/cases/117_convop_using_standard_ops/putAvailabilitySubscriptionConvOp.test]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**